extension/wallet_lib/src/wallet.rs

```diff
@@ -88,7 +87,7 @@ async fn current<'n, 'w>(
         _ => bail!("wallet with name '{}' is currently not loaded", name),
     };
 
-    Ok(MutexGuard::map(guard, |w| w.as_mut().unwrap_throw()))
+    Ok(MutexGuard::map(guard, |w| w.as_mut().unwrap()))
```

Comment:
:question: Why did this change?

Reply:
`unwrap_throw` is a feature of wasm_bindgen which was removed.